src/moin/converter/_tests/test_markdown_in_out.py

```diff
@@ -0,0 +1,146 @@
+# Copyright: 2008 MoinMoin:BastianBlank
+# Copyright: 2010 MoinMoin:DmitryAndreev
+# Copyright: 2018 MoinMoin:RogerHaase - modified test_moinwiki_in_out.py for markdown
+# License: GNU GPL v2 (or any later version), see LICENSE.txt for details.
+
+"""
+MoinMoin - Tests for markdown->DOM->markdown using markdown_in and markdown_out converters
+
+TODO: Failing tests are commented out and need to be fixed.
```

Comment:
did you look at the commented stuff? would be cool to have new stuff without old TODOs.